wolfssl/wolfcrypt/settings.h

```diff
@@ -31,6 +31,11 @@
     extern "C" {
 #endif
 
+/* include the options.h file ourselves for OpenVPN */
+#ifdef ENABLE_CRYPTO_WOLFSSL
+#include <wolfssl/options.h>
```

Comment:
After talking to Todd, he doesn't want to see options.h included in settings.h
Customers don't use options.h may have this on and cause complications.
Please take this out.

Reply:
What if I create a really obscure define in OpenVPN like `ENABLE_INCLUDE_OPTIONS_H_FOR_OPENVPN`? This would not be mistakenly added in other customer projects and OpenVPN would not have to include `options.h`.

Reply:
Nope! :-)
It annoys me that we don't have options.h in settings.h.
We can't for the case where someone builds OpenVPN for embedded and also has their own user_settings.h file for wolfssl.

Reply:
I talked to Todd about how to accomplish this. We decided to protect the include with a specific define (`ENABLE_INCLUDE_OPTIONS_H_FOR_OPENVPN`) and to make sure that this is never defined while building wolfSSL (`#error` out in `internal.c`). I also added documentation in `settings.h` to explain the exact use case for this include. On OpenVPN's side I added a configure script option `--disable-wolfssl-options-h` to allow for defining a custom `user_settings.h` file.